commands/psql.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const co = require('co')
+const cli = require('heroku-cli-util')
+
+function * run (context, heroku) {
+  const fetcher = require('../lib/fetcher')(heroku)
+  const psql = require('../lib/psql')
+
+  const {app, args, flags} = context
+
+  let db = yield fetcher.database(app, args.database)
+  cli.console.error(`--> Connecting to ${cli.color.addon(db.addon.name)}`)
```

Comment:
@dickeyxxx After some more digging I found that we can always assume an addon.  The old ruby code does not support DATABASE_URL when there is no matching attachment.  It basically pulls the config vars and tries to match up DATABASE_URL with the attachment config vars and if it matches, then it uses that attachment.